inc/core/acf_blocks/sites/cleanbc/project-query.php

```diff
@@ -88,7 +90,38 @@
 
                 <h4><?php echo esc_html( get_the_title( $project->ID ) ); ?></h4>
 
-                <?php echo wp_kses_post( get_the_content( null, false, $project->ID ) ); ?>
+                <?php echo wp_kses_post( get_the_content( null, true, $project->ID ) ); ?>
+                
+                <?php
+                $dom1 = new \DOMDocument();
```

Comment:
yuk!!! php dom stuff. is this the best way of doing this? Maybe a comment of why you are using php dom, as no one in my mind should be using this awful thing.  Could Javascript serve the same purpose?

Reply:
I don't disagree. This was the fastest solution I could come up with to grab the Wordpress meta comments associated with blocks. I added an aria label to buttons which works fine when the block is rendered directly but when it is queried as part of get_the_content it does not. It does dump the WP meta comment along side the base button block so this code grabs that and generates a div with a data attribute that can be used to inject an aria label into the button. It's not elegant and given time I will see if I can come up with a pure JS node traversal solution.